_inc/client/plans/single-product-backup.jsx

```diff
@@ -216,19 +278,21 @@ class SingleProductBackupBody extends React.Component {
 						/>
 					) ) }
 				</div>
-				{ upgradeLinks &&
-					upgradeLinks[ selectedBackupType ] &&
-					upgradeTitles[ selectedBackupType ] && (
-						<div className="single-product-backup__upgrade-button-container">
-							<Button
-								href={ upgradeLinks[ selectedBackupType ] }
-								onClick={ this.handleUpgradeButtonClick( selectedBackupType ) }
-								primary
-							>
-								{ upgradeTitles[ selectedBackupType ] }
-							</Button>
-						</div>
-					) }
+				<p>
+					<em>
+						<ProductSavings
```

Comment:
If we want to keep this here, we should make sure that the user can switch to monthly. Otherwise, it's weird that they're saving if they don't have the other option (monthly) available to them 😉

Reply:
I guess, let's go with it for now, but consider implementing yearly/monthly billing switch in wp-admin for backups.